src/main/java/com/learning/browsers/DriverModule.java

```diff
@@ -0,0 +1,29 @@
+package com.learning.browsers;
+
+import com.google.inject.Exposed;
+import com.google.inject.PrivateModule;
+import com.google.inject.Provides;
+import com.google.inject.Singleton;
+import com.learning.configuration.PropertiesReader;
+import org.openqa.selenium.WebDriver;
+
+public class DriverModule extends PrivateModule {
+
+  @Provides
+  @Exposed
+  @Singleton
+  public PropertiesReader providePropertiesReader() {
+    return new PropertiesReader();
+  }
+
+  @Provides
+  @Exposed
+  @Singleton
+  public WebDriver provideWebDriver(PropertiesReader propertiesReader) {
+    return new WebDriverFactory(propertiesReader).getWebDriver();
+  }
```

Comment:
I think this is enough in case of WebDriver Singleton however I will implement singleton check in `WebDriverFactory` at Friday for learning purpose.